spring-pulsar-spring-boot-autoconfigure/src/main/resources/META-INF/spring/org.springframework.boot.autoconfigure.AutoConfiguration.imports

```diff
@@ -0,0 +1 @@
+org.springframework.pulsar.autoconfigure.PulsarAutoConfiguration
```

Comment:
Remove manual import file once we go to SB M5. 

Had to revert b188436ab5db56faf713d39593a67e2999864c2e (auto-generated auto-config imports file only in SB snapshot)

Reply:
Created https://github.com/spring-projects-experimental/spring-pulsar/issues/110 to go to next milestone and undo these temp changes made in this current PR.

Reply:
Created https://github.com/spring-projects-experimental/spring-pulsar/issues/110 to go to next milestone and undo these temp changes made in this current PR.

Reply:
Created https://github.com/spring-projects-experimental/spring-pulsar/issues/110 to go to next milestone and undo these temp changes made in this current PR.